### DISCUSSION TITLE
Update Custom Colors in WYSIWYG ACF issue

### DISCUSSION THREAD

**Discussion by garryburch** [flyntwp/flynt#412] [Q&A] [answered]
In tinyMce.php of flynt-1.4-0 I am having issue adding custom color choice to this

![Screenshot_20211117_124118](https://user-images.githubusercontent.com/24479568/142264183-659462ea-fe3f-4c50-9280-90fab52d370e.png)

I'm not sure where to start inside this wp-content/themes/flynt-1.4.0/inc/tinyMce.php

Thanks for any help

**Comment by steffenbew** [comment, marked as answer]
@garryburch I think you might add a [textcolor_map](https://www.tiny.cloud/docs-4x/plugins/textcolor/) array to the [config array](https://github.com/flyntwp/flynt/blob/master/inc/tinyMce.php#L71-L198).

**Reply by garryburch** [reply]
Thank you
